store.go

```diff
@@ -2498,7 +2499,15 @@ func (s *store) DeleteContainer(id string) error {
 			gcpath := filepath.Join(s.GraphRoot(), middleDir, container.ID)
 			wg.Add(1)
 			go func() {
-				errChan <- os.RemoveAll(gcpath)
+				var err error
+				for attempts := 0; attempts < 50; attempts++ {
+					err = os.RemoveAll(gcpath)
+					if err == nil || !errors.Is(err, unix.EBUSY) {
```

Comment:
I think we need move `erros.Is(err, unix.EBUSY)` with a wrapper to unix specific package with `// +build linux freebsd darwin` preferably https://github.com/containers/storage/blob/main/pkg/system/syscall_unix.go and return no-op for non-unix.

Reply:
thanks, fixed now